sources.md

```diff
@@ -0,0 +1,10 @@
+# sources
+I will attach a dataset that I found in the wikidata
+
+[SNIRH data]https://snirh.pt/snirh/_dadosbase/site/simplex.php?OBJINFO=DADOS&FILTRA_BACIA=138&FILTRA_COVER=920123704&FILTRA_SITE=920685260
+
+From SNIRH database we have data for monthly mean air temperature and monthly precipitation from 01/10/1980 to 01/02/2022.
```

Comment:
Try to group this paragraph with the one above. You can use a header (##) or an item list (`-`)

Reply:
# SNIRH database#
-  we have data for monthly mean air temperature and monthly precipitation from 01/10/1980 to 01/02/2022.
- With these data we can understand the correlation between precipitation and air temperature and answer the main question of client has.
- It seems good and we could choose the monthly average temperature and monthly precipitation value